lib/ex_webrtc/rtp_receiver.ex

```diff
@@ -213,8 +213,10 @@ defmodule ExWebRTC.RTPReceiver do
   @spec get_stats(receiver(), non_neg_integer()) :: [map()]
   def get_stats(receiver, timestamp) do
     Enum.map(receiver.layers, fn {rid, layer} ->
+      id = if(rid == nil, do: receiver.track.id, else: "#{receiver.track.id}, rid: #{rid}")
```

Comment:
I would favor using `:` and no spaces as puting spaces and commas in id isn't a common practice

```suggestion
      id = if(rid == nil, do: receiver.track.id, else: "#{receiver.track.id}:#{rid}")
```